src/main/scala/pipelines/images/imagenet/ImageNetSiftLcsFV.scala

```diff
@@ -48,7 +48,7 @@ object ImageNetSiftLcsFV extends Serializable with Logging {
         new BatchPCATransformer(convert(csvread(new File(fname)), Float).t)
       case None =>
         val pca = sampledColumns andThen
-            (ColumnPCAEstimator(numPCADesc), trainingData)
```

Comment:
Yeah, let's just go ahead and do it - in general I think we want to move toward the higher level expression of these pipelines, which includes a bunch of benefits like showing up in integration tests (maybe) and it encourages us to eat our own dog food and really use the optimizer.